helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJob.java

```diff
@@ -193,21 +196,12 @@ void deploy() {
         }
       }
 
-      // Wait for jobs to have container ids
```

Comment:
Ah so this logic was moved to https://github.com/spotify/helios/pull/376/files#diff-d935303eb7db0e249425458704c02455R246?